k8s/charts/seaweedfs/templates/admin/admin-statefulset.yaml

```
@@ -0,0 +1,317 @@
+          {{- toYaml . | nindent 10 }}
+        {{- end }}
+      spec:
+        accessModes: [ "ReadWriteOnce" ]
+        storageClassName: {{ .Values.admin.logs.storageClass }}
+        resources:
+          requests:
+            storage: {{ .Values.admin.logs.size }}
+    {{- end }}
+  {{- end }}
+{{- end }}
+
```

YAMLlint (1.37.1)

[warning] too many blank lines (1 > 0)

(empty-lines)